src/hooks/index.ts

```diff
@@ -13,8 +9,14 @@ sbvrUtils.addPureHook('all', 'all', 'all', {
 	},
 });
 
-readdirAsync(__dirname + '/resources').each(initScript => {
-	if (/\.ts$/.test(initScript)) {
-		return require(__dirname + '/resources/' + initScript);
-	}
-});
+import * as _api_key from './resources/api_key';
```

Comment:
Just to check, these imports are still in the generated js right?

Reply:
They're not… good catch. Fixed.